cmd/dapr.go

```diff
@@ -39,11 +39,7 @@ var RootCmd = &cobra.Command{
 
 ===============================
 Distributed Application Runtime`,
-	Run: func(cmd *cobra.Command, args []string) {
-		if versionFlag {
-			printVersion()
-		}
```

Comment:
I think adding return inside the block and having command print usage outside the block should solve this issue.

Reply:
👍，Thanks